autoload/classpath.vim

```diff
@@ -0,0 +1,37 @@
+let s:entry = "\t<classpathentry kind=\"<kind>\" path=\"<path>\"/>"
+let s:entry_sourcepath = "\t<classpathentry kind=\"<kind>\" path=\"<path>\" sourcepath=\"<src>\"/>"
+
+" Creates a .classpath file and fills it with dependency entries
+function! classpath#generateClasspath() abort
+  let l:classes = extend(gradle#classPaths(), android#classPaths())
```

Comment:
Recently I removed android#classPaths() function. The gradle#classPaths() now returns all libs including android ones. This should do the same:

```
let l:classes = gradle#classPaths()
```